examples/label_studio_annotation/README.md

```diff
@@ -157,7 +194,8 @@ label-studio start -p 8093
 You should install the relevant integrations:
 
 ```shell
-zenml integration install label_studio pytorch s3 pillow
+# assuming label_studio was installed earlier
+zenml integration install pytorch s3 pillow
```

Comment:
isn't the old version clearer? or does calling this integration install break something by e.g. installing a different version?

Reply:
You can't install all of them as part of the same `zenml integration install` command... it can't resolve it.

Reply:
damn that's not ideal 😅

Reply:
They are supposedly at work to fix this in a big way. They even wrote to let us know in our slack and said they'd update the integration accordingly, so 🤷‍♂️